projects/kit/components/stepper/stepper.component.ts

```diff
@@ -72,12 +73,20 @@ export class TuiStepperComponent {
 
     @Input('activeItemIndex')
     public set activeIndex(index: number) {
+        this.itemIndex = index;
+
         if (this.steps.get(index)?.nativeElement) {
             this.activeItemIndex = index;
```

Comment:
Why do we only update internal index if there is such a step? That would cause out of sync situations when external index is A and internally we have B. Shouldn't we update it all the time and scroll to the element only when it's there?

Also what if we change from setter to `ngOnChanges`? Would content children be processed by then?

Reply:
fixed